views/partials/header.ejs

```diff
@@ -0,0 +1,22 @@
+<h1>The Jobs EJS Application</h1>
+<%if (user) { %>
+   <p>User <%= user.name %> is logged on.</p>
+   <form method="POST" action="/sessions/logoff">
```

Comment:
Add this field <input type="hidden" name="_csrf" value="<%= _csrf %>"> after this line. It's required for csrf validator.